src/thunderbots/software/util/parameter/dynamic_parameters.cpp

```diff
@@ -71,4 +71,29 @@ namespace Util::DynamicParameters
         Parameter<double> angular_sensitivity("xbox_demo_angular_sensitivity", 1.0);
     }  // namespace XBoxControllerDemo
 
+    namespace NetworkInput
+    {
+        namespace Filter
+        {
+            namespace PartileFilter
+            {
+                Parameter<int32_t> PARTICLE_FILTER_NUM_CONDENSATIONS(
```

Comment:
We should make these `lowercase_camel_case` to be consistent with the other parameters we have here.